Remove ENABLE_LAYER_BASED_SVG_ENGINE build flag

donny-dont opened the pull request:
#### 9d4b24463c6eadcb58801aafd4eb8fe730d1f03d
<pre>
Remove ENABLE_LAYER_BASED_SVG_ENGINE build flag
<a href="https://bugs.webkit.org/show_bug.cgi?id=271738">https://bugs.webkit.org/show_bug.cgi?id=271738</a>

Reviewed by Michael Catanzaro and Said Abou-Hallawa.

All ports are building with `ENABLE_LAYER_BASED_SVG_ENGINE` as `ON`. Remove the
build option and use the runtime option for controlling use.

* Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml:
* Source/WTF/wtf/PlatformEnable.h:
* Source/WTF/wtf/PlatformEnableCocoa.h:
* Source/WebCore/accessibility/AccessibilityRenderObject.cpp:
* Source/WebCore/dom/Element.cpp:
* Source/WebCore/loader/ImageLoader.cpp:
* Source/WebCore/loader/cache/CachedImage.cpp:
* Source/WebCore/page/LocalFrameView.cpp:
* Source/WebCore/page/SettingsBase.cpp:
* Source/WebCore/page/SettingsBase.h:
* Source/WebCore/rendering/ReferencedSVGResources.cpp:
* Source/WebCore/rendering/ReferencedSVGResources.h:
* Source/WebCore/rendering/RenderBox.cpp:
* Source/WebCore/rendering/RenderElement.cpp:
* Source/WebCore/rendering/RenderElement.h:
* Source/WebCore/rendering/RenderLayer.cpp:
* Source/WebCore/rendering/RenderLayer.h:
* Source/WebCore/rendering/RenderLayerBacking.cpp:
* Source/WebCore/rendering/RenderLayerInlines.h:
* Source/WebCore/rendering/RenderLayerModelObject.cpp:
* Source/WebCore/rendering/RenderLayerModelObject.h:
* Source/WebCore/rendering/RenderObject.cpp:
* Source/WebCore/rendering/RenderObject.h:
* Source/WebCore/rendering/RenderTreeAsText.cpp:
* Source/WebCore/rendering/RenderView.cpp:
* Source/WebCore/rendering/TransformOperationData.cpp:
* Source/WebCore/rendering/svg/RenderSVGBlock.cpp:
* Source/WebCore/rendering/svg/RenderSVGBlock.h:
* Source/WebCore/rendering/svg/RenderSVGContainer.cpp:
* Source/WebCore/rendering/svg/RenderSVGContainer.h:
* Source/WebCore/rendering/svg/RenderSVGEllipse.cpp:
* Source/WebCore/rendering/svg/RenderSVGEllipse.h:
* Source/WebCore/rendering/svg/RenderSVGForeignObject.cpp:
* Source/WebCore/rendering/svg/RenderSVGForeignObject.h:
* Source/WebCore/rendering/svg/RenderSVGGradientStop.cpp:
* Source/WebCore/rendering/svg/RenderSVGHiddenContainer.cpp:
* Source/WebCore/rendering/svg/RenderSVGHiddenContainer.h:
* Source/WebCore/rendering/svg/RenderSVGImage.cpp:
* Source/WebCore/rendering/svg/RenderSVGImage.h:
* Source/WebCore/rendering/svg/RenderSVGInline.cpp:
* Source/WebCore/rendering/svg/RenderSVGInline.h:
* Source/WebCore/rendering/svg/RenderSVGInlineText.cpp:
* Source/WebCore/rendering/svg/RenderSVGModelObject.cpp:
* Source/WebCore/rendering/svg/RenderSVGModelObject.h:
* Source/WebCore/rendering/svg/RenderSVGPath.cpp:
* Source/WebCore/rendering/svg/RenderSVGPath.h:
* Source/WebCore/rendering/svg/RenderSVGRect.cpp:
* Source/WebCore/rendering/svg/RenderSVGRect.h:
* Source/WebCore/rendering/svg/RenderSVGResourceClipper.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceClipper.h:
* Source/WebCore/rendering/svg/RenderSVGResourceContainer.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceContainer.h:
* Source/WebCore/rendering/svg/RenderSVGResourceGradient.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceGradient.h:
* Source/WebCore/rendering/svg/RenderSVGResourceGradientInlines.h:
* Source/WebCore/rendering/svg/RenderSVGResourceLinearGradient.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceLinearGradient.h:
* Source/WebCore/rendering/svg/RenderSVGResourceLinearGradientInlines.h:
* Source/WebCore/rendering/svg/RenderSVGResourceMarker.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceMarker.h:
* Source/WebCore/rendering/svg/RenderSVGResourceMasker.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceMasker.h:
* Source/WebCore/rendering/svg/RenderSVGResourcePaintServer.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourcePaintServer.h:
* Source/WebCore/rendering/svg/RenderSVGResourcePattern.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourcePattern.h:
* Source/WebCore/rendering/svg/RenderSVGResourcePatternInlines.h:
* Source/WebCore/rendering/svg/RenderSVGResourceRadialGradient.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceRadialGradient.h:
* Source/WebCore/rendering/svg/RenderSVGResourceRadialGradientInlines.h:
* Source/WebCore/rendering/svg/RenderSVGRoot.cpp:
* Source/WebCore/rendering/svg/RenderSVGRoot.h:
* Source/WebCore/rendering/svg/RenderSVGShape.cpp:
* Source/WebCore/rendering/svg/RenderSVGShape.h:
* Source/WebCore/rendering/svg/RenderSVGShapeInlines.h:
* Source/WebCore/rendering/svg/RenderSVGText.cpp:
* Source/WebCore/rendering/svg/RenderSVGText.h:
* Source/WebCore/rendering/svg/RenderSVGTextPath.cpp:
* Source/WebCore/rendering/svg/RenderSVGTransformableContainer.cpp:
* Source/WebCore/rendering/svg/RenderSVGTransformableContainer.h:
* Source/WebCore/rendering/svg/RenderSVGViewportContainer.cpp:
* Source/WebCore/rendering/svg/RenderSVGViewportContainer.h:
* Source/WebCore/rendering/svg/SVGBoundingBoxComputation.cpp:
* Source/WebCore/rendering/svg/SVGBoundingBoxComputation.h:
* Source/WebCore/rendering/svg/SVGContainerLayout.cpp:
* Source/WebCore/rendering/svg/SVGContainerLayout.h:
* Source/WebCore/rendering/svg/SVGInlineTextBox.cpp:
* Source/WebCore/rendering/svg/SVGInlineTextBox.h:
* Source/WebCore/rendering/svg/SVGLayerTransformComputation.h:
* Source/WebCore/rendering/svg/SVGLayerTransformUpdater.h:
* Source/WebCore/rendering/svg/SVGPaintServerHandling.h:
* Source/WebCore/rendering/svg/SVGRenderSupport.cpp:
* Source/WebCore/rendering/svg/SVGRenderTreeAsText.cpp:
* Source/WebCore/rendering/svg/SVGRenderingContext.cpp:
* Source/WebCore/rendering/svg/SVGResourcesCache.cpp:
* Source/WebCore/rendering/svg/SVGResourcesCycleSolver.cpp:
* Source/WebCore/rendering/svg/SVGRootInlineBox.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGForeignObject.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGForeignObject.h:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResource.cpp:
* Source/WebCore/rendering/updating/RenderTreeBuilder.cpp:
* Source/WebCore/rendering/updating/RenderTreeBuilderSVG.cpp:
* Source/WebCore/rendering/updating/RenderTreeBuilderSVG.h:
* Source/WebCore/rendering/updating/RenderTreeUpdater.cpp:
* Source/WebCore/style/StyleAdjuster.cpp:
* Source/WebCore/svg/SVGAElement.cpp:
* Source/WebCore/svg/SVGAnimateMotionElement.cpp:
* Source/WebCore/svg/SVGCircleElement.cpp:
* Source/WebCore/svg/SVGClipPathElement.cpp:
* Source/WebCore/svg/SVGDefsElement.cpp:
* Source/WebCore/svg/SVGElement.cpp:
* Source/WebCore/svg/SVGEllipseElement.cpp:
* Source/WebCore/svg/SVGFEImageElement.cpp:
* Source/WebCore/svg/SVGForeignObjectElement.cpp:
* Source/WebCore/svg/SVGGElement.cpp:
* Source/WebCore/svg/SVGGeometryElement.cpp:
* Source/WebCore/svg/SVGGradientElement.cpp:
* Source/WebCore/svg/SVGGraphicsElement.cpp:
* Source/WebCore/svg/SVGImageElement.cpp:
* Source/WebCore/svg/SVGLineElement.cpp:
* Source/WebCore/svg/SVGLinearGradientElement.cpp:
* Source/WebCore/svg/SVGLocatable.cpp:
* Source/WebCore/svg/SVGMarkerElement.cpp:
* Source/WebCore/svg/SVGMaskElement.cpp:
* Source/WebCore/svg/SVGPathElement.cpp:
* Source/WebCore/svg/SVGPatternElement.cpp:
* Source/WebCore/svg/SVGPolyElement.cpp:
* Source/WebCore/svg/SVGRadialGradientElement.cpp:
* Source/WebCore/svg/SVGRectElement.cpp:
* Source/WebCore/svg/SVGSVGElement.cpp:
* Source/WebCore/svg/SVGSwitchElement.cpp:
* Source/WebCore/svg/SVGSymbolElement.cpp:
* Source/WebCore/svg/SVGUseElement.cpp:
* Source/WebCore/svg/graphics/SVGImage.cpp:
* Source/cmake/OptionsGTK.cmake:
* Source/cmake/OptionsMac.cmake:
* Source/cmake/OptionsPlayStation.cmake:
* Source/cmake/OptionsWPE.cmake:
* Source/cmake/OptionsWin.cmake:
* Source/cmake/WebKitFeatures.cmake:
* Tools/Scripts/webkitperl/FeatureList.pm:

Canonical link: <a href="https://commits.webkit.org/276817@main">https://commits.webkit.org/276817@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/fe57868f4929a79e00805d4dc05ff22f000a387f

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/45726 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/24853 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/48307 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/48395 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/41761 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/48032 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/29126 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/22249 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/37439 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/46304 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/21929 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/39452 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/18591 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/19329 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/40542 "Passed tests") | [✅ 🛠 wpe-skia](https://ews-build.webkit.org/#/builders/52/builds/3770 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/38956 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/42035 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/40881 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/50147 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/45197 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/20717 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/17230 "Found 1 new test failure: imported/w3c/web-platform-tests/html/cross-origin-opener-policy/coep-redirect.https.html (failure)") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/44548 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/22022 "Built successfully") | | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/43396 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/10160 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/22380 "Built successfully") | | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/52351 "Built successfully") | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/21710 "Built successfully") | | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/10708 "Passed tests") | 
<!--EWS-Status-Bubble-End-->

mcatanzaro reviewed (approved):
Looks good. This should never have used a build flag in the first place, since there are no external dependencies.

A more interesting question is whether to enable LBSE when running layout tests, and how long to keep the runtime flag around. That's probably for @nikolaszimmermann to decide, though, and is certainly a separate problem from removing the build flag.

shallawa reviewed (approved):
This change looks good to me also.

Some of the LBSE code is not guarded by `LAYER_BASED_SVG_ENGINE`, for example `isLegacyRenderSVGRoot()` and `isRenderSVGRoot()` and their siblings in RenderElement.h. So it makes sense to be consistent and remove the LBSE compilation guard completely.

Removing the `Legacy` should be straightforward once we decide to do so. All the legacy renderers are in one separate directory. So I think all we need to do is removing this directory and all the references to the classes in this directory.

donny-dont committed:
Remove ENABLE_LAYER_BASED_SVG_ENGINE build flag
https://bugs.webkit.org/show_bug.cgi?id=271738

Reviewed by Michael Catanzaro and Said Abou-Hallawa.

All ports are building with `ENABLE_LAYER_BASED_SVG_ENGINE` as `ON`. Remove the
build option and use the runtime option for controlling use.

* Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml:
* Source/WTF/wtf/PlatformEnable.h:
* Source/WTF/wtf/PlatformEnableCocoa.h:
* Source/WebCore/accessibility/AccessibilityRenderObject.cpp:
* Source/WebCore/dom/Element.cpp:
* Source/WebCore/loader/ImageLoader.cpp:
* Source/WebCore/loader/cache/CachedImage.cpp:
* Source/WebCore/page/LocalFrameView.cpp:
* Source/WebCore/page/SettingsBase.cpp:
* Source/WebCore/page/SettingsBase.h:
* Source/WebCore/rendering/ReferencedSVGResources.cpp:
* Source/WebCore/rendering/ReferencedSVGResources.h:
* Source/WebCore/rendering/RenderBox.cpp:
* Source/WebCore/rendering/RenderElement.cpp:
* Source/WebCore/rendering/RenderElement.h:
* Source/WebCore/rendering/RenderLayer.cpp:
* Source/WebCore/rendering/RenderLayer.h:
* Source/WebCore/rendering/RenderLayerBacking.cpp:
* Source/WebCore/rendering/RenderLayerInlines.h:
* Source/WebCore/rendering/RenderLayerModelObject.cpp:
* Source/WebCore/rendering/RenderLayerModelObject.h:
* Source/WebCore/rendering/RenderObject.cpp:
* Source/WebCore/rendering/RenderObject.h:
* Source/WebCore/rendering/RenderTreeAsText.cpp:
* Source/WebCore/rendering/RenderView.cpp:
* Source/WebCore/rendering/TransformOperationData.cpp:
* Source/WebCore/rendering/svg/RenderSVGBlock.cpp:
* Source/WebCore/rendering/svg/RenderSVGBlock.h:
* Source/WebCore/rendering/svg/RenderSVGContainer.cpp:
* Source/WebCore/rendering/svg/RenderSVGContainer.h:
* Source/WebCore/rendering/svg/RenderSVGEllipse.cpp:
* Source/WebCore/rendering/svg/RenderSVGEllipse.h:
* Source/WebCore/rendering/svg/RenderSVGForeignObject.cpp:
* Source/WebCore/rendering/svg/RenderSVGForeignObject.h:
* Source/WebCore/rendering/svg/RenderSVGGradientStop.cpp:
* Source/WebCore/rendering/svg/RenderSVGHiddenContainer.cpp:
* Source/WebCore/rendering/svg/RenderSVGHiddenContainer.h:
* Source/WebCore/rendering/svg/RenderSVGImage.cpp:
* Source/WebCore/rendering/svg/RenderSVGImage.h:
* Source/WebCore/rendering/svg/RenderSVGInline.cpp:
* Source/WebCore/rendering/svg/RenderSVGInline.h:
* Source/WebCore/rendering/svg/RenderSVGInlineText.cpp:
* Source/WebCore/rendering/svg/RenderSVGModelObject.cpp:
* Source/WebCore/rendering/svg/RenderSVGModelObject.h:
* Source/WebCore/rendering/svg/RenderSVGPath.cpp:
* Source/WebCore/rendering/svg/RenderSVGPath.h:
* Source/WebCore/rendering/svg/RenderSVGRect.cpp:
* Source/WebCore/rendering/svg/RenderSVGRect.h:
* Source/WebCore/rendering/svg/RenderSVGResourceClipper.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceClipper.h:
* Source/WebCore/rendering/svg/RenderSVGResourceContainer.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceContainer.h:
* Source/WebCore/rendering/svg/RenderSVGResourceGradient.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceGradient.h:
* Source/WebCore/rendering/svg/RenderSVGResourceGradientInlines.h:
* Source/WebCore/rendering/svg/RenderSVGResourceLinearGradient.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceLinearGradient.h:
* Source/WebCore/rendering/svg/RenderSVGResourceLinearGradientInlines.h:
* Source/WebCore/rendering/svg/RenderSVGResourceMarker.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceMarker.h:
* Source/WebCore/rendering/svg/RenderSVGResourceMasker.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceMasker.h:
* Source/WebCore/rendering/svg/RenderSVGResourcePaintServer.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourcePaintServer.h:
* Source/WebCore/rendering/svg/RenderSVGResourcePattern.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourcePattern.h:
* Source/WebCore/rendering/svg/RenderSVGResourcePatternInlines.h:
* Source/WebCore/rendering/svg/RenderSVGResourceRadialGradient.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceRadialGradient.h:
* Source/WebCore/rendering/svg/RenderSVGResourceRadialGradientInlines.h:
* Source/WebCore/rendering/svg/RenderSVGRoot.cpp:
* Source/WebCore/rendering/svg/RenderSVGRoot.h:
* Source/WebCore/rendering/svg/RenderSVGShape.cpp:
* Source/WebCore/rendering/svg/RenderSVGShape.h:
* Source/WebCore/rendering/svg/RenderSVGShapeInlines.h:
* Source/WebCore/rendering/svg/RenderSVGText.cpp:
* Source/WebCore/rendering/svg/RenderSVGText.h:
* Source/WebCore/rendering/svg/RenderSVGTextPath.cpp:
* Source/WebCore/rendering/svg/RenderSVGTransformableContainer.cpp:
* Source/WebCore/rendering/svg/RenderSVGTransformableContainer.h:
* Source/WebCore/rendering/svg/RenderSVGViewportContainer.cpp:
* Source/WebCore/rendering/svg/RenderSVGViewportContainer.h:
* Source/WebCore/rendering/svg/SVGBoundingBoxComputation.cpp:
* Source/WebCore/rendering/svg/SVGBoundingBoxComputation.h:
* Source/WebCore/rendering/svg/SVGContainerLayout.cpp:
* Source/WebCore/rendering/svg/SVGContainerLayout.h:
* Source/WebCore/rendering/svg/SVGInlineTextBox.cpp:
* Source/WebCore/rendering/svg/SVGInlineTextBox.h:
* Source/WebCore/rendering/svg/SVGLayerTransformComputation.h:
* Source/WebCore/rendering/svg/SVGLayerTransformUpdater.h:
* Source/WebCore/rendering/svg/SVGPaintServerHandling.h:
* Source/WebCore/rendering/svg/SVGRenderSupport.cpp:
* Source/WebCore/rendering/svg/SVGRenderTreeAsText.cpp:
* Source/WebCore/rendering/svg/SVGRenderingContext.cpp:
* Source/WebCore/rendering/svg/SVGResourcesCache.cpp:
* Source/WebCore/rendering/svg/SVGResourcesCycleSolver.cpp:
* Source/WebCore/rendering/svg/SVGRootInlineBox.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGForeignObject.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGForeignObject.h:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResource.cpp:
* Source/WebCore/rendering/updating/RenderTreeBuilder.cpp:
* Source/WebCore/rendering/updating/RenderTreeBuilderSVG.cpp:
* Source/WebCore/rendering/updating/RenderTreeBuilderSVG.h:
* Source/WebCore/rendering/updating/RenderTreeUpdater.cpp:
* Source/WebCore/style/StyleAdjuster.cpp:
* Source/WebCore/svg/SVGAElement.cpp:
* Source/WebCore/svg/SVGAnimateMotionElement.cpp:
* Source/WebCore/svg/SVGCircleElement.cpp:
* Source/WebCore/svg/SVGClipPathElement.cpp:
* Source/WebCore/svg/SVGDefsElement.cpp:
* Source/WebCore/svg/SVGElement.cpp:
* Source/WebCore/svg/SVGEllipseElement.cpp:
* Source/WebCore/svg/SVGFEImageElement.cpp:
* Source/WebCore/svg/SVGForeignObjectElement.cpp:
* Source/WebCore/svg/SVGGElement.cpp:
* Source/WebCore/svg/SVGGeometryElement.cpp:
* Source/WebCore/svg/SVGGradientElement.cpp:
* Source/WebCore/svg/SVGGraphicsElement.cpp:
* Source/WebCore/svg/SVGImageElement.cpp:
* Source/WebCore/svg/SVGLineElement.cpp:
* Source/WebCore/svg/SVGLinearGradientElement.cpp:
* Source/WebCore/svg/SVGLocatable.cpp:
* Source/WebCore/svg/SVGMarkerElement.cpp:
* Source/WebCore/svg/SVGMaskElement.cpp:
* Source/WebCore/svg/SVGPathElement.cpp:
* Source/WebCore/svg/SVGPatternElement.cpp:
* Source/WebCore/svg/SVGPolyElement.cpp:
* Source/WebCore/svg/SVGRadialGradientElement.cpp:
* Source/WebCore/svg/SVGRectElement.cpp:
* Source/WebCore/svg/SVGSVGElement.cpp:
* Source/WebCore/svg/SVGSwitchElement.cpp:
* Source/WebCore/svg/SVGSymbolElement.cpp:
* Source/WebCore/svg/SVGUseElement.cpp:
* Source/WebCore/svg/graphics/SVGImage.cpp:
* Source/cmake/OptionsGTK.cmake:
* Source/cmake/OptionsMac.cmake:
* Source/cmake/OptionsPlayStation.cmake:
* Source/cmake/OptionsWPE.cmake:
* Source/cmake/OptionsWin.cmake:
* Source/cmake/WebKitFeatures.cmake:
* Tools/Scripts/webkitperl/FeatureList.pm:

Canonical link: https://commits.webkit.org/276817@main
